pinc/stages.inc

```diff
@@ -15,40 +15,34 @@ include_once($relPath.'access_change_emails.inc'); // pp_access_change_email(),
 
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
-if ($site_supports_metadata)
-{
-
-new Stage(
-    'CA',
-    _('Clearance Approvals'),
-    array(),
-    'REQ-HUMAN',
-    '',
-    NULL, /* access_change_callback */
-    _('A qualified person must confirm that the work is in the public domain.'),
-    NULL,
-    "tools/site_admin/proj_approvals.php"
-);
-
+if ($site_supports_metadata) {
+    new Stage(
+        'CA',
+        _('Clearance Approvals'),
+        array(),
+        'REQ-HUMAN',
+        '',
+        NULL, /* access_change_callback */
+        _('A qualified person must confirm that the work is in the public domain.'),
+        NULL,
```

Comment:
`NULL` to lower case?

Reply:
I'm actually going to let the code formatter take care of those things (as well as a few more in this stanza). I just wanted to get them indented since the formatter didn't do what I wanted it to.